src/index.js

```diff
@@ -48,13 +47,20 @@ class TinderClient {
   }
 
   static createFromFacebookLogin({ emailAddress, password }) {
-    return generateAccessToken({
-      facebookEmailAddress: emailAddress,
-      facebookPassword: password,
-    }).then(accessToken => axios.create({
+    return axios({
+      method: 'GET',
+      url: 'https://b-graph.facebook.com/auth/login',
+      params: {
+        password: password,
+        email: emailAddress,
+        access_token: '350685531728|62f8ce9f74b12f84c123cc23437a4a32',
+        method: 'POST'
```

Comment:
Should the `method` property on [line 51](https://github.com/jaebradley/tinder-client/pull/92/files#diff-1fdf421c05c1140f6d71444ea2b27638R51) be removed?

Reply:
Ah, nvm, both this and the `access_token` value are explained by https://github.com/Schmavery/facebook-chat-api/pull/684#issuecomment-425624643